docs/modules/plugins/pages/plugin-azure-storage-blob.adoc

```diff
@@ -0,0 +1,112 @@
+= Azure Storage Blob Plugin
+
+The plugin provides functionality to interact with https://azure.microsoft.com/en-us/services/storage/blobs/[Azure Blob Storage]
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-azure-storage-blob', version: '{current-version}')
```

Comment:
I think we should stick to the naming of Azure services:
`vividus-plugin-azure-storage-account`

Blob storage is a part of storage account functionality